amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -65,8 +66,28 @@ import './styles.scss';
 const SERVER_ERROR_CODE = 500;
 const DASHBOARDS_PER_PAGE = 10;
 const TABLE_SOURCE = 'table_page';
-
-export interface StateFromProps {
+const SORT_CRITERIAS = {
+  usage: {
+    key: 'usage',
+    direction: 'desc',
+  },
+  sort_order: {
+    key: 'sort_order',
+    direction: 'asc',
+  },
+};
+const SORT_OPTIONS = [
+  {
+    name: 'Table Default',
+    type: 'sort_order',
+  },
+  {
+    name: 'Usage Count',
+    type: 'usage',
```

Comment:
Does this work right now as is? This looks like it works by needing a `usage` attribute on the column metadata, but usage is on the column stats.

Given this implementation -- unless I've missed some recent changes -- we will have to add logic in https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/api/utils/metadata_utils.py#L68 to pull the column usage object from the stats list (so that it doesnt show up on the stats UI), and add its value directly on the column object under the name `usage`.

Reply:
Oh, I do some rework of the data inside Column list o add that info into the data itself.

Reply:
Just noticed it. Left a comment about it below.